tests/src/test/scala/common/rest/WskRestOperations.scala

```diff
@@ -275,6 +275,7 @@ class RestActionOperations(implicit val actorSystem: ActorSystem)
     update: Boolean = false,
     web: Option[String] = None,
     websecure: Option[String] = None,
+    deleteOld: Boolean = true,
```

Comment:
- New options (deleteAll, deleteOld) need to be documented
- Swagger JSON file need to be updated

Reply:
document is updated, and I will update the swagger json soon